index.d.ts

```diff
@@ -109,6 +109,9 @@ declare class Agent implements Taggable, StartSpanFn {
 
   // Utils
   logger: Logger;
+
+  // Custom metrics
+  registerCustomMetric(name: string, callback: Function, labels?: Labels): void;
```

Comment:
I think it might be a bit more natural to always have the callback as the final parameter, and instead overload the function like:

```js
registerCustomMetric(name: string, callback: Function): void;
registerCustomMetric(name: string, labels?: Labels, callback: Function): void;
```

That way the identity of the metric – be it name, or name + labels – always comes first. WDYT?

Reply:
It's indeed more natural. Updated.